src/login_nopam.c

```diff
@@ -289,7 +289,7 @@ static const char *resolve_hostname (const char *string)
 
 /* from_match - match a host or tty against a list of tokens */
 
-static bool from_match (const char *tok, const char *string)
+static bool from_match (char *tok, const char *string)
```

Comment:
This one doesn't write, no?  Am I missing something?

Reply:
Ahhh, it's for compatibility with the function pointer.  Sorry for the noise.

Reply:
You are correct, it doesn't write. But if it's const, then it does not match the signature required by list_match. So ... unfortunately we are forced to do that here.

Reply:
WG14 has a paper proposing relaxing these rules for C2y.  It basically proposes allowing certain implicit conversions between function pointers that are known to be safe.  It was uploaded the day after we discussed this.
<https://www.open-std.org/jtc1/sc22/wg14/www/docs/n3449.pdf>

Coincidence or not?  :-)